tools/style.sh

```diff
@@ -23,8 +23,8 @@ git diff --stat
 # Render documents
 Rscript -e "if (!require('rmarkdown')) install.packages('rmarkdown')"
 Rscript -e "if (!require('roxygen2')) install.packages('roxygen2')"
+Rscript -e 'roxygen2::roxygenize()'
```

Comment:
We should use `tools/document.R` here to use our custom roclets.

Reply:
Actually rebuilding the docs might be a bit too much to do on every PR after styling. 
If styling changes compiled code, the package will need to be re-installed to re-build the docs, adding maybe more than 10 minutes to the CI.

I'm leaning towards prefering to have a special PR command that runs styling instead of running it at every commit to the PR.

Reply:
Yes, I'd run it if `contains(github.event.head_commit.message, '[style]')` & skip on PR. What do you think?

Reply:
I like this style of PR commands: https://github.com/r-lib/actions/blob/v2-branch/examples/pr-commands.yaml
Where we just need to comment `/style` in the PR to trigger the workflow. This way we don't need to push dummy commits in order to trigger styling.

Reply:
Sounds good to me.